CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ created.
 
 ### Added
 
+- ([#428](https://github.com/microsoft/InnerEye-DeepLearning/issues/428). Fixed bug #428: ensure that dataset.csv is checked
```

Comment:
this should reference the PR, not the issue.

Reply:
Changed to: 
- ([#428](https://github.com/microsoft/InnerEye-DeepLearning/issues/428). Added a sanity check to ensure there are no
  missing channels, nor missing files. If missing channels in the csv file or filenames associated with channels are
  incorrect, pipeline exits with error report before running training or inference.